week2/Week2/src/week2/EvenOrOdd.java

```diff
@@ -0,0 +1,32 @@
+
+package week2;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author sara
+ */
+public class EvenOrOdd {
+
+    public static void printNum(int number){
+        if(number%2==0){
```

Comment:
Please use white spaces to make your code more readable and easier to understand.
Try using the formatter in IntelliJ : Code-> Reformate code or (CTRL+ALT+L)